### PR TITLE
EN-26 - CN - Simplified and Traditional Chinese language options

### PR DESCRIPTION
**Pull request by fedelocci**
Cards

- EN-50
- EN-51

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/com/agiletec/aps/system/services/lang/ISO_639 -1_langs.xml [hunk 1]
```diff
@@ -553,6 +553,10 @@
 		<descr>Chinese</descr>
 	</Lang>
 	<Lang>
+		<code>zhs</code>
+		<descr>Chinese Simplified</descr>
+	</Lang>
```
Comment (nicpuddu) [inline engine/src/main/java/com/agiletec/aps/system/services/lang/ISO_639 -1_langs.xml]: it is Simplified Chinese, not Chinese Simplified

we also need to amend the other one to Traditional Chinese

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**